### DISCUSSION TITLE
c-lightning plugins

### DISCUSSION THREAD

**Discussion by jaonoctus** [btcpayserver/btcpayserver#2430] [Technical Support] [answered]
How could we install c-lightning plugins? Such as [rebalance](https://github.com/lightningd/plugins/tree/master/rebalance)

**Comment by Kukks** [comment, marked as answer]
> How could we install c-lightning plugins? Such as [rebalance](https://github.com/lightningd/plugins/tree/master/rebalance)

You can, by creating a plugins directory in the clightning/bitcoin data directory.  If you ran the docker install on a Linux machine, it is most likely at `/var/lib/docker/volumes/generated_clightning_bitcoin_datadir/_data/bitcoin`

**Comment by Kukks** [comment]
https://github.com/btcpayserver/btcpayserver-docker/pull/518
